system/LINK/etc/default/evm

```diff
@@ -26,3 +30,4 @@ export QPID_SSL_CERT_DB=/etc/qpid/certs
 [[ -s /opt/rh/postgresql92/enable ]] && source /opt/rh/postgresql92/enable
 [[ -s /opt/rh/nodejs010/enable ]] && source /opt/rh/nodejs010/enable
 [[ -s /etc/default/evm_productization ]] && source /etc/default/evm_productization
+[[ -s /opt/rh/httpd24/enable ]] && source /opt/rh/httpd24/enable
```

Comment:
Wondering if this should be before evm_productization...in the end it probably doesn't even matter.